.github/docker-sync/regsync.yml

```diff
@@ -35,3 +36,24 @@ sync:
   - source: ghcr.io/projectnessie/nessie
     target: docker.io/projectnessie/nessie
     type: repository
+  # GC Tool
+  - source: ghcr.io/projectnessie/nessie-gc-unstable
+    target: quay.io/projectnessie/nessie-gc-unstable
+    type: repository
+    tags:
```

Comment:
You can omit the tags here. We needed those for Nessie to not mirror super-old versions.
Since you're already touching this file - can you change the regex for Nessie server above to something like `0[.][789]\\d[.].*|[123][.].*` - to include everything like `0.7[0-9]....` and newer - and `1.xxx` as well.